README.md

```diff
@@ -154,11 +186,41 @@ produce early errors because of recursive application of the first rule.
 - Let _targetName_ be ? Get(_target_, `"name"`).
 - If Type(_targetName_) is not `String`, then let _targetName_ be the empty string.
 - Let _status_ be ? SetFunctionName(_F_, _targetName_, `"bound"`).
+- Assert: _status_ is never used.
 - Return _F_.
 
 
 ----
 
+    BindExpression:
+        LeftHandSideExpression :: new
+
+- Let _targetReference_ be the result of evaluating _LeftHandSideExpression_.
+- Assert: IsSuperReference(_targetReference_) is `false` if Type(_targetReference_) is Reference.
```

Comment:
We need to remove this, since the left operand can certainly be a super reference:

```js
super.Foo::new
```